src/adapter/src/coord/sequencer.rs

```diff
@@ -1075,17 +1076,16 @@ impl<S: Append + 'static> Coordinator<S> {
         task::spawn(
             || format!("sink_connection_ready:{}", sink.from),
             async move {
-                internal_cmd_tx
-                    .send(Message::SinkConnectionReady(SinkConnectionReady {
-                        session,
-                        tx,
-                        id,
-                        oid,
-                        result: sink_connection::build(connection_builder, id, connection_context)
-                            .await,
-                        compute_instance,
-                    }))
-                    .expect("sending to internal_cmd_tx cannot fail");
+                // It is not an error for sink connections to become ready after `internal_cmd_rx` is dropped.
```

Comment:
Is there any cleanup we need to do here with removing the dummy sink from the catalog, or will that happen on the next boot up?

Reply:
I 100% have no idea! :D

Reply:
Probably the same thing will happen on the next boot-up that would have happened had we panicked here, though. So if we would be in an inconsistent state, it seems like we would have been in one previously. But I am not the expert!